shell/client/shell.html

```diff
@@ -1287,8 +1324,6 @@ <h3 id="app-stats">Apps</h3>
   <form id="admin-settings-form">
     <p>Server Title: <input type="text" name="serverTitle" value="{{serverTitle}}"><br>
       <span class="details">The name of this server, for use when communicating with users. Example: "Bob's Sandstorm Server"</span></p>
-    <p>Return Address: <input type="email" name="returnAddress" value="{{returnAddress}}"><br>
```

Comment:
It's also used as the last unit in the page `<title>`.  Which is sufficiently non-email-related that I think it doesn't rightly live under email configuration.

Sharing emails should migrate (Soon™) to sending as (From:) the sharer's display name and selected verified email address.  I guess this setting would still be relevant for login and transactional emails, but I think that it applies more broadly than email.

Perhaps the optimal design is to have an *additional* setting for the name of the email sender in the email config, which if empty, defaults to the value of Server Title (and explains all this).